### ISSUE TITLE
[WCAG] appBar Reflow - Icon Buttons become inaccessible on Zoom

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/GeoView#3565]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

In full-screen mode with browser set to 1280px width and zoomed to 400%, appBar IconButtons can become inaccessible; they overlap and can't be accessed using mouse/keyboard.

<img width="1105" height="1383" alt="Image" src="https://github.com/user-attachments/assets/06607213-b461-41c1-b2d7-acf5d13c9eed" />

<img width="563" height="640" alt="Image" src="https://github.com/user-attachments/assets/5343c234-4eda-43b7-a5ac-4e8639530eb7" />

### Expected Behavior

All IconButtons should be accessible.
